Zscaler/zscaler-zia/tests/test_event_casb.json

```diff
@@ -0,0 +1,30 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Zscaler Internet Access [BETA]",
+        "dialect_uuid": "dc0f339f-5dbe-4e68-9fa0-c63661820941"
+      }
+    },
+    "message": "{\"sourcetype\": \"zscalernss-casb\",\"event\": {\"login\": \"john.doe@example.org\",\"tenant\": \"example\",\"object_type\": \"0\",\"applicationname\": \"ONEDRIVE\",\"object_name_1\": \"sanity2022-09-04 00-06.pdf\",\"object_name_2\": \"Maverick\"}}"
+  },
+  "expected": {
+    "message": "{\"sourcetype\": \"zscalernss-casb\",\"event\": {\"login\": \"john.doe@example.org\",\"tenant\": \"example\",\"object_type\": \"0\",\"applicationname\": \"ONEDRIVE\",\"object_name_1\": \"sanity2022-09-04 00-06.pdf\",\"object_name_2\": \"Maverick\"}}",
```

Comment:
Can you extract the `applicationname` to `network.application` and `object_name_1` to `file.name`?